src/Integration.UnitTests/Binding/CSharpVBBindingOperation.WriterTests.cs

```diff
@@ -196,16 +200,25 @@ public void ProjectBindingOperation_QueueWriteProjectLevelRuleSet_ProjectHasExis
                 numRules: 0,
                 includes: new[] { newSolutionRuleSetInclude }
             );
-            var dotNetConfig = new CSharpVBBindingConfig(expectedRuleSet, newSolutionRuleSetPath);
+
+            var csharpVbConfig = CreateCSharpVbBindingConfig(newSolutionRuleSetPath, expectedRuleSet);
 
             // Act
-            string actualPath = testSubject.QueueWriteProjectLevelRuleSet(projectFullPath, ruleSetName, dotNetConfig, existingProjectRuleSetPath);
+            string actualPath = testSubject.QueueWriteProjectLevelRuleSet(projectFullPath, ruleSetName, csharpVbConfig, existingProjectRuleSetPath);
 
             // Assert
             this.ruleSetFS.AssertRuleSetsAreEqual(actualPath, expectedRuleSet);
             actualPath.Should().Be(existingProjectRuleSetPath, "Expecting the rule set to be updated");
         }
 
+        private static CSharpVBBindingConfig CreateCSharpVbBindingConfig(string newSolutionRuleSetPath, RuleSet expectedRuleSet)
```

Comment:
extracted to function as it's now more complicated to do "new CSharpVBBindingConfig"